src/util.rs

```diff
@@ -100,7 +109,24 @@ pub fn start_daemon<I, S>(program: &str, args: I) -> io::Result<()>
         I: IntoIterator<Item = S>,
         S: AsRef<OsStr>,
 {
-    Command::new(program).args(args).spawn().map(|_| ())
+    // As per issue 2128 the decision was made to make this call as
+    // similar to the Unix behaviour as possible.
+    //
+    // Setting all the I/O handles to null and setting the
+    // CREATE_NEW_PROCESS_GROUP and CREATE_NO_WINDOW has the effect
+    // that console applications will run without opening a new
+    // console window.
+    //
+    // If visible output is desirable the DETACHED_PROCESS flag could
+    // be used instead of CREATE_NO_WINDOW.
+    Command::new(program)
+        .args(args)
+        .stdin(Stdio::null())
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
```

Comment:
Shouldn't this be the default anyways? What happens when all this `Stdio::null()` mappings are removed?

Reply:
According to https://doc.rust-lang.org/std/process/struct.Command.html#method.new, for `.spawn()` the default is to inherit handles rather than null. This is also true for linux - but the `::libc::daemon(1, 0)` call will set all of the stdio handles to null in that case.

Reply:
You are absolutely correct, I just tested it again to confirm it and I must have remembered that incorrectly. Stdout of a child will get redirected to the stdout of the parent by default.